cdap-examples/UserProfiles/src/main/java/co/cask/cdap/examples/profiles/Profile.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.examples.profiles;
+
+/**
+ * The profile of a user. It contains the user id, name and email address,
+ * as well the last login time and the time of his last activity.
```

Comment:
The profile of a user. It contains the user id, name, email address,
last login time, and time of their last activity.